### DISCUSSION TITLE
Need to add "Dedication" page right after abstract in Overleaf template

### DISCUSSION THREAD

**Discussion by MOmarFarooq** [ustctug/ustcthesis#349] [Q&A]
Hello Everyone,

Please guide me to include the "Dedication" page in overleaf version of USTC thesis right after abstract. By default it is not showing there. Please find the attachment for your convenience.
![DedicationGithub](https://user-images.githubusercontent.com/20922494/167254671-55b7734d-494f-4bda-bb89-386ad6e42b4d.png)

Kindly response me on that.

Thank you

**Comment by zepinglee** [comment]
The dedication page is not specified in the thesis requirements, so you may write in whatever format you want. An example is as follows.

```tex
\begin{titlepage}
  \null\vfil
  \begin{center}
    {\Large\textbf{Dedication}\par}
    \vskip 3em%
    A special feeling of gratitude to my loving father
    
    \vskip 3cm%
    Some other words
  \end{center}
  \vfil\null
\end{titlepage}
```

Besides, I suggest putting the dedication page ahead of abstract because the page number starts at the abstract page.

**Reply by MOmarFarooq** [reply]
Thank you so much!
